### PR TITLE
feat(frontend): handle delete for tags 

### PR DESCRIPTION
**Pull request by MariePrandstetter**
#4709 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/component/Tags/Tag.jsx [hunk 1]
```diff
@@ -1,41 +1,47 @@
...
-        {props.name}
-      </label>
+    <div className='tagList__list__item__wrapper'>
+      <i className='fas fa-play tagList__list__item__triangle'></i>
+      <div className='tagList__list__item'>
+        <div className='tagList__list__item__circle' />
+        <label
+          className='tagList__list__item__info'
+          title={props.name}
+        >
```
Comment (raphj) [inline frontend_lib/src/component/Tags/Tag.jsx:17]: This label is missing a `htmlFor` attribute. Turn it into span if having it as a label is not relevant anymore.

File: frontend_lib/src/component/Tags/Tag.jsx [hunk 2]
```diff
@@ -1,41 +1,47 @@
...
-  onClickCheckbox: () => { }
-}
-
 Tag.propTypes = {
-  checked: PropTypes.bool,
   name: PropTypes.string.isRequired,
-  tagId: PropTypes.number.isRequired,
-  onClickCheckbox: PropTypes.func
+  isContent: PropTypes.bool,
+  onClickDeleteTag: PropTypes.func,
```
Comment (raphj) [inline frontend_lib/src/component/Tags/Tag.jsx:39]: Maybe isRequired?

**Review by raphj — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/Tags/Tag.jsx [hunk 3]
```diff
@@ -1,41 +1,47 @@
...
-      />
-      <label
-        htmlFor={'checkbox-tag-' + props.tagId}
-        className='tagList__list__item__info'
-        title={props.name}
-      >
-        {props.name}
-      </label>
+    <div className='tagList__list__item__wrapper'>
+      <i className='fas fa-play tagList__list__item__triangle'></i>
```
Comment (gbonaspetti) [inline frontend_lib/src/component/Tags/Tag.jsx:11]:
```suggestion
      <i className='fas fa-play tagList__list__item__triangle' />
```

**Review by raphj — APPROVED** [review]

**Comment by charline-rageade** [comment]
Tested on branch

**Pull request merged**